projects/packages/google-fonts-provider/src/class-google-fonts-provider.php

```diff
@@ -154,6 +154,15 @@ public function get_css() {
 	private function build_collection_api_urls() {
 		$font_families_urls = array();
 
+		/**
+		 * Filters the Google Fonts API URL.
+		 *
+		 * @since 0.4.0
```

Comment:
```suggestion
		 * @since $$next-version$$
```

Reply:
It's a nice to have, and a way to ensure you'll always have the right version number without having to think about it. The string gets replaced by a real version number when the plugin gets released.